src/babashka/neil/meta.clj

```diff
@@ -0,0 +1,5 @@
+(ns babashka.neil.meta)
+
+;; This is a fallback if the var hasn't already been set earlier by a build
+;; script, such as when neil is called as a library instead of a script.
+(defonce version :version-not-set)
```

Comment:
why `defonce` instead of `def`?

Reply:
@borkdude: If `def` is used here, it will overwrite the `version` set in `babashka.neil.meta` by `bb gen-script`. The purpose of this namespace is to allow the code to work when used outside of `bb gen-script`, such as in the REPL. The `:version-not-set` value should only appear if there is no existing `version`.

Reply:
ok

Reply:
I considered adding code to load the version here for local dev or REPL usage. I ultimately decided it wasn't worth it since it will add complexity to figure out where to load the `neil` version from in these other contexts. We have a `--deps-file` option but that's for the project version, not the `neil` verision.